File: Livros.Server/Models/VendaLivro.cs
using System;
using System.Collections.Generic;

namespace Livros.Server.Models;

public partial class VendaLivro
{
    public int CodVl { get; set; }

    public int LivroCodl { get; set; }

    public int VendaCodV { get; set; }

    public int Quantidade { get; set; }

    public decimal ValorUnitario { get; set; }

    public decimal ValorDesconto { get; set; }

    public decimal ValorTotal { get; set; }

    public bool Ativo { get; set; }

    public virtual Livro LivroCodlNavigation { get; set; } = null!;

    public virtual Vendum VendaCodVNavigation { get; set; } = null!;
}

File: livros.client/src/app/models/VendaLivro.ts
import { Livro } from './Livro';
import { Venda } from './Venda';

export class VendaLivro {
  codVl!: number;
  livro_Codl!: Number;
  venda_CodV!: number;
  quantidade!: number;
  valorUnitario!: number;
  valorDesconto!: number;
  valorTotal!: number;
  ativo!: boolean;
  livro!: Livro[];
  venda?: Venda;
}
